.github/workflows/setup-conda.yml

```diff
@@ -0,0 +1,31 @@
+# Verify that the setup-conda action works
+
+name: Validate setup-conda action
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  conda:
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest, windows-latest]
+        conda: [anaconda, miniconda]
+
+    runs-on: ${{ matrix.os }}
+    name: ${{ matrix.os }}-${{ matrix.conda }}
+
+    steps:  
+    - name: Setup conda
+      uses: iiasa/actions/setup-conda@main
+      with:
+        installer: ${{ matrix.conda }}
```

Comment:
I'm not sure if naming this `installer` might interfere with the command "installer" in the shell later. Should we rename it?